ProjectSettings/MemorySettings.asset

```diff
@@ -0,0 +1,35 @@
+%YAML 1.1
```

Comment:
Unsure why this file is a new addition - perhaps just never checked in before so probably makes sense.